modules/rostests/winetests/msvcrt/CMakeLists.txt

```diff
@@ -34,13 +38,14 @@ if(CMAKE_C_COMPILER_ID STREQUAL "GNU" OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
     endif()
 endif()
 
-set_module_type(msvcrt_winetest win32cui)
-add_importlibs(msvcrt_winetest advapi32 msvcrt kernel32)
-target_link_libraries(msvcrt_winetest oldnames)
-
 if(MSVC)
-    add_importlibs(msvcrt_winetest ntdll)
+    # 'swscanf' : unknown type field character '\xe1' in format specifier
```

Comment:
Is it a new warning showing up only due to the changes of this PR? or?
Because in the PR patch, the only instances of `\xe1` I see are in the `test__strnicmp_l` that doesn't use swscanf :thinking:

Reply:
`C:\ReactOS\reactos\modules\rostests\winetests\msvcrt\scanf.c(426): warning C4476: 'swscanf' : unknown type field character '\xe1' in format specifier
`
https://github.com/reactos/reactos/blob/bb2334066e26499b583ec13b6618a082114c4d36/modules/rostests/winetests/msvcrt/scanf.c#L426

Reply:
not sure how `\x1234` or `%\x1234` translate into `\xe1` specifier, but OK ...